### PR TITLE
Feature/events application

### PR DESCRIPTION
**Pull request by Rumbie-Mudzie7**
The following procedures were implemented in the private-events milestone:

-  Users were allowed to create events and then manage user signups.
-  Users were allowed to send invitations and parties.
-  Events could take place at a specific date and at a location.
-  many-to-many relationship associations between models.
-  belongs-to relationship association on models that belong to the User model.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  end
+
+  # GET /users/1
+  # GET /users/1.json
+  def show
+    # @user = User.find(params[:id])
+    @user = current_user
+    @created_events = @user.created_events
+    # @previously_attended_events = current_user.attended_events.previous_events
+    # @upcoming_events = current_user.attended_events.upcoming_events
```
Comment (uche-inyama) [inline app/controllers/users_controller.rb]: Delete these commented lines of code.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  validates :email, presence: true, uniqueness: true
+  validates :name, presence: true
+
+  # has_many :event_attendings, foreign_key: "attendee_id"
+  # has_many :created_events, foreign_key: "creator_id", class_name: "Event"
+  # has_many :attended_events, through: :event_attendings
```
Comment (uche-inyama) [inline app/models/user.rb]: Remove these commented lines of code.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes required

Hi team how is it going   :wave: 

Let's work on some additional features and necessary updates:

* [x] The event's show page should also display the list of those attending the event ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/events_attendees_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

* [x] The user's show page should display events the user has created and events that the user is attending, separated in past and upcoming ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

* [x] There are commented lines of code, in one of your models, I have left a comment below.

* [x] Remove if/else statement in your view files, refactor them in their corresponding helper files.

* [x] There are no RSpec tests, kindly implement your test with RSpec. I can see you wrote your test with the Minitest approach.

Make these changes, when you are done submit another form for a code review.

#### ``` Have a great day ``` 


``` Uchechkwu Henry Inyama```

**Review by rOluochKe — APPROVED** [review]
Hello @Rumbie-Mudzie7 and @ClaytonSiby 
### You got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**